frontend/src/components/wrappers/Button.jsx

```diff
@@ -30,13 +30,16 @@ export default function Button({
 }
 
 Button.propTypes = {
-  linkTo: PropTypes.string.isRequired,
+  linkTo: PropTypes.string,
   onClick: PropTypes.func.isRequired,
-  className: PropTypes.string.isRequired,
-  children: PropTypes.string.isRequired,
+  className: PropTypes.string,
+  children: PropTypes.node,
   color: PropTypes.string,
 };
 
 Button.defaultProps = {
+  linkTo: null,
   color: 'bg-primary',
+  className: null,
+  children: null,
```

Comment:
```suggestion
  linkTo: undefined,
  color: 'bg-primary',
  className: undefined,
  children: undefined,
```
i apologize again for the inaccurate suggestion :pray:

Reply:
yeah no problem